content/docs/operating/security.md

```diff
@@ -0,0 +1,143 @@
+silences.
+
+Where notifications are sent to is determined by the configuration file. With
+certain templating setups it is possible for notifications to end up at an
+alert-defined destination. For example if notifications use an alert label as
+the destination email address, anyone who can send alerts to the alertmanager
+can send notifications to any email address.
+
+## Pushgateway
+
+Any user with access to the Pushgateway HTTP endpoint can create, modify and
+delete the metrics contained within. As the Pushgateway is usually scraped with
+`honor_labels` enabled, this means anyone with access to the Pushgateway can
+create any time series in Prometheus.
+
+## Exporters
+
+Exporters generally only talk to one configured instance with a preset set of
+commands/requests, which cannot be expanded via their HTTP endpoint.
+
+There are also exporters such as the SNMP and Blackbox exporters that take
+their targets from URL parameters. Thus anyone with HTTP access to these
+exporters can make them send requests to arbitrary endpoints. As they also
+support client-side authentication, this could lead to a leak of secrets such
+as HTTP Basic Auth passwords or SNMP community strings. Challenge-response
+authentication mechanisms such as TLS are not affected by this.
+
+## Client Libraries
+
+Client libaries are intended to be included in users' applications.
+
+If using a client libary provided HTTP handler, it should not be possible for
+malicious requests that reach that handler to cause issues beyond those
+resulting from additional load and failed scrapes.
+
+## Authentication/Authorisation/Encryption
+
+Prometheus and its components do not provide any server-side
+authentication, authorisation or encryption. If you require this, it is
+recommended to use a reverse proxy.
+
+Various Prometheus components support client-side authentication and
+encryption. If TLS client support is offered, there is often also an option
+called `insecure_skip_verify` which skips SSL verification.
+
+## Secrets
+
+Non-secret information or fields may be available via the HTTP API and/or logs.
+
+In Prometheus, metadata retrieved from service discovery is not considered
+secret. Throughout the Prometheus system, metrics are not considered secret.
+
+Fields containing secrets in configuration files (marked explicitly as such in
+the documentation) will not be exposed in logs or via the HTTP API. Secrets
+should not be placed in other configuration fields, as it is common for
+components to expose their configuration over their HTTP endpoint.
+
+Secrets from other sources used by dependencies (e.g. the `AWS_SECRET_KEY`
+enviroment vabiable as used by EC2 service discovery) may end up exposed due to
+code outside of our control or due to functionality that happens to expose
+wherever it is stored.
+
+## Denial of Service
+
+While there are some mitigations in place for excess load or expensive queries.
```

Comment:
A sentence starting with "While ..." (subordinate clause) needs a main clause following it (after a comma).

Just omit the "While" and it's correct :)